Makefile

```
@@ -6,7 +6,8 @@ MODULES_PATH := /lib/modules
 MODULES_RELEASE_PATH := ${MODULES_PATH}/${KERNEL_RELEASE}
 MODULES_ORDER_LIST := lunatik/lunatik.ko kernel/zfs/zfs.ko # needed for Ubuntu
 MODULES_ORDER_FILE := ${MODULES_RELEASE_PATH}/modules.order
-MODULES_BUILD_PATH ?= ${MODULES_RELEASE_PATH}/build
+KERNEL_BUILD_DIR = ${MODULES_RELEASE_PATH}/build
+MODULES_BUILD_PATH ?= ${KERNEL_BUILD_DIR}
```

```suggestion
MODULES_BUILD_PATH = ${MODULES_RELEASE_PATH}/build
BTF_INSTALL_PATH ?= ${MODULES_BUILD_PATH}
```
I think it would be clearer to state this var is used to install "btf" here..

---

I think we dont want to override MODULES_BUILD_PATH if it's passed from the command line ... that's why I use `?=` instead of `=`.

---

ah.. got you, sorry.. so, I think this would comprehend both concerns:
```
BTF_INSTALL_PATH = ${MODULES_RELEASE_PATH}/build
MODULES_BUILD_PATH ?= ${BTF_INSTALL_PATH}
```
that is, just `s/KERNEL_BUILD_DIR/BTF_INSTALL_PATH/g` in you original patch..

what do you think?

---

It makes sense. i will apply it